ada-project-docs/testing.md

```diff
@@ -0,0 +1,17 @@
+## Tests
+
+We will need to complete part of all of some of the tests for this project. If a test is incomplete, it will raise an `Exception`. You should comment out the `Exception` when implementing the test. 
```

Comment:
Small typo:
```suggestion
We will need to complete part – or all – of some of the tests for this project. If a test is incomplete, it will raise an `Exception`. You should comment out the `Exception` when implementing the test. 
```